src/test/java/de/retest/web/it/websites/ReTestTest.java

```diff
@@ -0,0 +1,64 @@
+package de.retest.web.it.websites;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+import de.retest.recheck.Recheck;
+import de.retest.recheck.RecheckImpl;
+
+/*
+ * Simple recheck-web showcase for a Chrome-based integration test. See other *IT classes for more examples.
+ */
+public class ReTestTest {
+
+	private WebDriver driver;
+	private Recheck re;
+
+	@Before
+	public void setup() {
+		// If ChromeDriver (http://chromedriver.chromium.org/downloads/) is not in your PATH, uncomment this and point to your installation.
+		//		System.setProperty( "webdriver.chrome.driver", "path/to/chromedriver" );
+
+		final ChromeOptions opts = new ChromeOptions();
+		opts.addArguments(
+				// Enable headless mode for faster execution.
+				"--headless",
+				// Use Chrome in container-based Travis CI enviroment (see https://docs.travis-ci.com/user/chrome#Sandboxing).
+				"--no-sandbox",
+				// Fix window size for stable results.
+				"--window-size=1200,800" );
+		driver = new ChromeDriver( opts );
+
+		// Use the default implementation.
+		re = new RecheckImpl();
+	}
+
+	@Test
+	public void index() throws Exception {
+		// Set the file name of the Golden Master.
+		re.startTest( "ReTest" );
+
+		driver.get( "https://retest.de/" );
```

Comment:
Problem here is that this creates an "outside" dependency. When we change the retest website, the build (then master!) starts to fail...
So either add lokal HTML files or do not add this test, as this is reflected in existing tests already.